packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -62,6 +62,12 @@ export class BrowserTool extends BaseBrowserTool {
     this.viewport = { width: config.width, height: config.height };
     this.testContext = config.testContext;
 
+    // Update active page reference to a newly opened tab
+    this.page.context().on("page", async (newPage) => {
```

Comment:
This fixes the issue reported, where the AI didn't have the new tab, after clicking on the email link, as active.

@m2rads do you think this could cause any possible side effects?